### ISSUE TITLE
When the value is 0 or false, the editor has no content

### ISSUE THREAD

**Issue by nineaiyu** [bestkolobok/vue3-jsoneditor#18]
![image](https://github.com/user-attachments/assets/57d0ec62-2aea-447a-84e5-9d88c3761178)

```
  <json-editor
    ref="editor"
    v-model:json="value"
    :navigationBar="false"
    :onRenderMenu="handleRenderMenu"
    :fullWidthButton="false"
    height="400"
    mode="text"
    @change="handleChange"
  />
```

**Comment by bestkolobok** [comment]
Hi @nineaiyu ,

I fixed this bug in version 2.11.3

**Comment by nineaiyu** [comment]
Thank you very much.
